pyiode/iode_database/abstract_database.pyx

```diff
@@ -1350,11 +1350,11 @@ cdef class _AbstractDatabase:
         >>> # 1. default relax to 1.0
         >>> scalars["a0"] = 0.1
         >>> scalars["a0"]
-        Scalar(0.1, 1, nan)
```

Comment:
wouldn't it be better if the repr used NA instead of na (to match the capitalization of the imported constant)?